src/operations/txbuild.js

```diff
@@ -429,12 +429,13 @@ function makePreorder(fullyQualifiedName: string,
     return Promise.all([network.getUTXOs(preorderAddress), network.getFeeRate(), preorderPromise])
       .then(([utxos, feeRate, preorderSkeleton]) => {
         const txB = bitcoinjs.TransactionBuilder.fromTransaction(preorderSkeleton, network.layer1)
+        txB.setVersion(1)
```

Comment:
By default transactions now all have `version = 2` when they are created. Functionally this has no effect on what blockstack.js actually does, but it does make some tests fail due to different signatures and different transaction sizes resulting from the different signatures. There are tests that make assumptions about the transaction sizes and they fail without setting `version = 1`.

To resolve this you can either change the assumption in the test or add this line. I opted to add this line to demonstrate that all tests pass without modification. Perhaps a reviewer would feel more confident in changing the tests and removing this from the production code.